channels/audin/client/audin_main.c

```diff
@@ -607,14 +626,14 @@ static UINT audin_on_close(IWTSVirtualChannelCallback* pChannelCallback)
 	AUDIN_CHANNEL_CALLBACK* callback = (AUDIN_CHANNEL_CALLBACK*) pChannelCallback;
 	AUDIN_PLUGIN* audin = (AUDIN_PLUGIN*) callback->plugin;
```

Comment:
Why is the check 

	if (!callback || !data)
		return ERROR_INVALID_PARAMETER;

done in audin_on_data_received and not in audin_on_close?

should all functions be checked which can be called from outside, or via callback ?